mangaki/mangaki/utils/anidb.py

```diff
@@ -97,25 +97,63 @@ def anime_category(self) -> Category:
 
     @cached_property
     def lang_map(self) -> Dict[str, ExtLanguage]:
-        ext_langs = (
-            ExtLanguage.objects.filter(source='anidb')
-            .select_related('lang')
-        )
+        ext_langs = ExtLanguage.objects.filter(source='anidb').select_related('lang')
+        return {ext.lang.code: ext for ext in ext_langs}
 
-        return {
-            ext.lang.code: ext for ext in ext_langs
-        }
+    @cached_property
+    def role_map(self) -> Dict[str, Role]:
+        return {role.slug: role for role in Role.objects.all()}
 
     @cached_property
     def unknown_language(self) -> ExtLanguage:
         return ExtLanguage.objects.get(source='anidb', ext_lang='x-unk')
 
+    def get_xml(self, anidb_aid: int):
+        """Return the XML file for an anime from AniDB given its AniDB ID"""
+        anidb_aid = int(anidb_aid)
```

Comment:
We supposed that this function only receives integer, no need to fix user's errors.
It's better to signal them early.

As `get_xml` is type-hinted for `int`, let it crash if we receive something else.